spec/models/manageiq/providers/ansible_tower/configuration_manager/refresh_parser_spec.rb

```diff
@@ -15,6 +15,7 @@
   let(:all_job_templates) { (1..2).collect { |i| AnsibleTowerClient::JobTemplate.new("id" => i, "name" => "template#{i}", "description" => "description#{i}", "extra_vars" => "some_json_payload") } }
 
   it "#configuration_manager_inv_to_hashes" do
+    allow_any_instance_of(AnsibleTowerClient::JobTemplate).to receive(:survey_spec).and_return('some_hash_payload')
```

Comment:
I wasn't sure how to force output on a method of an instance - since all_job_templates returns an array and I need to stub survey_spec return data on an instance.